### DISCUSSION TITLE
Bidirectional relationship doesn't auto-connect when created

### DISCUSSION THREAD

**Discussion by CPapadim** [pods-framework/pods#6758] [Ideas]
I had a pod with a standard relationship, which contains a bunch of data.

Next, I wanted to make that relationship bidirectional.  However, I'm not able to connect the already existing data in a bidirectional way unless I re-save each of the 5000+ posts.  For already-existing data, the relationship is still one way even after I change the field to bidirectional.  For new data the field is bidirectional.

Is there a way to accomplish this so that when a bidirectional relationship is created, the existing data is updated and shows up without manually re-saving thousands of pieces of content?

**Comment by Brian-Milnes** [comment]
Yes, I've noticed this too. So +1

**Comment by sc0ttkclark** [comment]
I think we can tackle this one in our relationship work in 2.8

**Comment by sc0ttkclark** [comment]
Needs retest in 2.8

**Comment by pdclark** [comment]
@sc0ttkclark This can't be tested in 2.8, because the Bi-directional field does not populate with any fields with which to make a bi-directional relationship. See #6032 — needs resolution before #4644 can be tested.
Switching back to 2.7 resolves #6032.

**Comment by sc0ttkclark** [comment]
Once the other issue is resolved, this should be noted that it's not a guarantee that the enhancement this issue describes will work. I'm just interested to find out if it was fixed as a result of some of our other work. If not, we will punt this to 2.9 for backend work to be done.

**Comment by sc0ttkclark** [comment]
Ready for QA test @pdclark

**Comment by pdclark** [comment]
QA didn't pass — bi-directional field could be selected in 2.8, but fields did not auto-associate bi-directionally.
@sc0ttkclark

**Comment by sc0ttkclark** [comment]
Gotcha, we will move this to 2.9 and try to tackle it on the field save process.
